third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -106,9 +115,15 @@ std::vector<ComputationClient::ComputationPtr> PjRtComputationClient::Compile(
   std::vector<ComputationClient::ComputationPtr> computations;
 
   for (auto& instance : instances) {
+    PjRtDevice* pjrt_device = StringToPjRtDevice(instance.compilation_device);
     xla::ProgramShape program_shape =
         instance.computation.GetProgramShape().ValueOrDie();
     xla::CompileOptions compile_options;
+    // TODO(wcromar): set compile_options.argument_layouts, enable strict shapes
+    compile_options.executable_build_options.set_num_replicas(
```

Comment:
This is not necessary as it default to 1 anyway, but might be useful to explicitly set `num_partitions` to be specific about the compile & execution mode -- I think these two always go hand-to-hand.

Reply:
That's a good point. I'll hard-code one partition to be explicit about the assumption here.